freshdesk/v2/api.py

```diff
@@ -135,8 +135,12 @@ def list_tickets(self, **kwargs):
             url += "?filter=%s&" % filter_name
         else:
             url += "?"
-        page = kwargs.get("page", 1)
-        per_page = kwargs.get("per_page", 100)
+
+        if "updated_since" in kwargs:
```

Comment:
Could you please add a test and small line in the README under `list_ticket()` to show this is now supported? From looking at this I don't know what format it should be in.

Reply:
I have updated the PR just now. Can you please check if everything looks good?
Thanks!